add installers linux and winget (EN + PT) (#93)
ZupIT/docs-ritchie: content/pt-br/Primeiros Passos/Linux.md

## Passo 1: Rode o comando de instalação

### Última versão

Copie e cole o comando abaixo para executar no terminal:

```text
curl -fsSL https://commons-repo.ritchiecli.io/install.sh | bash
```

### Pacotes

> Para baixar uma versão ou um pacote específico, use as URLs abaixo no seu navegador, substituindo o campo `{VERSION}` de acordo com [a tag do repositório do projeto](https://github.com/ZupIT/ritchie-cli/tags):

Reviewer: ```suggestion
Se você quiser baixar uma versão ou um pacote específico, use as URLs abaixo no seu navegador. Substitua o campo `{VERSION}` de acordo com [**a tag do repositório do seu projeto**](https://github.com/ZupIT/ritchie-cli/tags):
```

#### Red Hat Package Manager

```url
https://commons-repo.ritchiecli.io/{VERSION}/installer/ritchie.rpm
```

#### Debian

```url
https://commons-repo.ritchiecli.io/{VERSION}/installer/ritchie.deb
```

#### Arch Linux

> O pacote `tar.gz` é disponível nesta página: [Arch Linux user repository](https://aur.archlinux.org/packages/ritchie-cli/) ([repositório de referência](https://github.com/avelino/ritchie-cli-archpack)).

Reviewer: ```suggestion
O pacote `tar.gz` é disponível nesta página: [**Arch Linux user repository**](https://aur.archlinux.org/packages/ritchie-cli/) ([**repositório de referência**](https://github.com/avelino/ritchie-cli-archpack)).
```

> O pacote `tar.gz` é disponível nesta página: [**Arch Linux user repository**](https://aur.archlinux.org/packages/ritchie-cli/) ([**repositório de referência**](https://github.com/avelino/ritchie-cli-archpack)).
> 

Reviewer: ```suggestion
O pacote `tar.gz` está disponível na página: [**Arch Linux user repository**](https://aur.archlinux.org/packages/ritchie-cli/) e veja o ([**repositório de referência**](https://github.com/avelino/ritchie-cli-archpack)).

```
